packages/fast-components-styles-msft/src/number-field/index.ts

```diff
@@ -16,12 +15,12 @@ const styles: ComponentStyles<NumberFieldClassNameContract, DesignSystem> = (
     return {
         numberField: {
             ...inputFieldStyles(designSystem),
-            height: density(defaultHeight)(designSystem),
-            minHeight: toPx(minHeight),
-            maxHeight: toPx(maxHeight),
+            height: height()(designSystem),
+            minHeight: minHeight()(designSystem),
+            maxHeight: maxHeight()(designSystem),
```

Comment:
FYI there is a utility called `localizeSpacing` that will split the string and re-arrange it automatically.

Also, do these actually need to be different from LTR to RTL? see `padding()` vs `padding(2)`

Reply:
Updated to `localizeSpacing`. The difference is because of the up and down arrows. It's somewhat temporary considering we want to stylize this component, but it's so they don't push all the way over.